src/api/v1beta1/dynakube_status.go

```diff
@@ -79,10 +96,50 @@ type VersionStatus struct {
 	LastUpdateProbeTimestamp *metav1.Time `json:"lastUpdateProbeTimestamp,omitempty"`
 }
 
+func (verStatus *VersionStatus) GetImageHash() string {
+	return verStatus.ImageHash
```

Comment:
Should `verStatus.ImageHash` be exported? If yes, getter is not necessary, if no, please change it to lower case.

Reply:
It must be exported unless you want me to add a setter as well:
![image](https://user-images.githubusercontent.com/16673172/152969016-c33071a7-8b98-427e-a6fc-d38ae250ecfe.png)

Alternatively, I can get rid of `.GetImageHash()` etc. and just access the exported fields - I don't have a strong preference here.

Reply:
For sake of consistency choose either of these: getters+setters or Exported field.
I think the latter is preferred in go world, less boilerplate.

Reply:
I'd prefer having getters without get in the name: [EffectiveGo#Getters](https://go.dev/doc/effective_go#Getters)

Reply:
Actually, the fields of `VersionStatus` must be exported because so that they can be (un)marshalled from/to JSON or YAML.

I simplified the interfaces a bit and renamed the remaining getters as suggested, thanks.